### ISSUE TITLE
Registration form not handled correctly when using activation mail in Contao 4.4

### ISSUE THREAD

**Issue by JulienSteininger** [contao/contao#996]
**Affected version(s)**
Contao 4.4.45

**Description** 
When registering with a mail that is already in use for another deactivated account the activation mail is not resent, no new user is registered, the form fields are not displayed and the submit button has no text. The problems are caused by this check happening too early so the rest of the function is not executed.

```
                // Check for a follow-up registration (see #7992)
                if ($this->reg_activate && \Input::post('email', true) != '' && ($objMember = \MemberModel::findUnactivatedByEmail(\Input::post('email', true))) !== null)
                {
                        $this->resendActivationMail($objMember);

                        return;
                }
```
Moving the check directly above the user creation (see below) solves most of the problems. The text of the submit button is still not displayed correctly however.
```
    // Create new user if there are no errors
                if (\Input::post('FORM_SUBMIT') == $strFormId && !$doNotSubmit)
                {
                        $this->createNewUser($arrUser);
                }
```

**How to reproduce**  
* Activate "Send activation e-mail" in the registration module
* register a member but do not activate the account yet
* Register another member using the same email.

**Comment by leofeyer** [comment]
I cannot reproduce this, because I get an "This entry already exists!" error.

<img width="626" alt="" src="https://user-images.githubusercontent.com/1192057/69332077-39859d00-0c56-11ea-82d7-e02a9c90dd8f.png">

**Comment by JulienSteininger** [comment]
Corrected version of the report here: #1258
